### PR TITLE
Add networth leaderboard position to hover lore

### PR DESCRIPTION
**Pull request by Soopyboo32**
image

wow so wicked its showing position on leaderboard!

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/ProfileViewer.java [hunk 1]
```diff
@@ -770,6 +771,29 @@ public String[] getCategories() {
...
+		 */
+		public long getSoopyNetworthLeaderboardPosition(){
+			return soopyNetworthLeaderboardPosition;
+		}
+
+		public boolean isProfileMaxSoopyNetworth(String profileName){
+			String highestProfileName = "";
+			long largestProfileNetworth = 0;
+
+			for(String pName : soopyNetworth.keySet()){
```
Comment (jani270) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/ProfileViewer.java:786]: Ctrl+Alt+L :trollface:

Comment (Ascynx) [reply src/main/java/io/github/moulberry/notenoughupdates/profileviewer/ProfileViewer.java]: :trollface:

**Comment by Ascynx** [comment]
much wow

**Comment by romangraef** [comment]
I think this is bugged, why in the screenshot is someone with 10000 coins networth placed number 2784

**Comment by jani270** [comment]
> I think this is bugged, why in the screenshot is someone with 10000 coins networth placed number 2784

surely

**Comment by jani270** [comment]
> I think this is bugged, why in the screenshot is someone with 10000 coins networth placed number 2784

(if you mean that unironically the person has 10b nw not 10k)

**Review by romangraef — DISMISSED** [review]

**Review by NopoTheGamer — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]
I'm looking forward to having the next PR in branch master4

**Pull request merged**